cmd/kubeapps-apis/plugins/resources/v1alpha1/server.go

```diff
@@ -33,6 +33,11 @@ import (
 	"github.com/kubeapps/kubeapps/cmd/kubeapps-apis/plugins/pkg/statuserror"
 )
 
+const (
+	DISCOVERY_CLIENT_QPS   = 50.0
+	DISCOVERY_CLIENT_BURST = 100
```

Comment:
Good catch! Would it make sense to have them as environment variables, defaulted to those values if not present? Don't remember if we are using env variables for this kind of setup. I was thinking in the cases of environments where K8s resources are scarce (e.g. Docker desktop extension or small clusters).

Reply:
+1 to provide these vars externally. I mean, we're already passing `QPS` and `burst` in the chart, so it is just a matter of using it, no?

Reply:
> It might even be a chance to replace the (growing) args to the grpc registration function with a struct so that we don't need to update the function signature each time.

+1, that would be nice and give flexibility to the registration process.

Reply:
OK, I've updated this PR, rebasing on #4372 so it's clear as an example how we can update the args to the plugin registration without having to modify other plugins necessarily. PTAL :)